lib/tasks/helpers/format.rb

```diff
@@ -81,9 +84,10 @@ def format_name(key)
     name = NAME_OVERRIDES[key]
     return name if name
 
-    key.split('_')
+    title = key.split('_')
       .each { |fragment| fragment[0] = fragment[0].upcase }
       .join(' ')
+    "#{title} [##{key.tr('_', '-')}]"
```

Comment:
The `tr` here doesn't make sense as-is. All underscores have already been removed given `'an_example'.split('_').join(' ')`. Did you intend to replace the spaces with hyphens?

```suggestion
    "#{title} [##{key.tr(' ', '-')}]"
```

Reply:
Hmm... I don't think so, the output turned out how I hoped it to.

The original key looks something like "application_logging". The return value I'm hoping for is "Application Logging [#application-logging]". 

`key` is the original argument. In `title`, the underscores are replaced with a space, and the first letter of each word is capitalized `application_logging` => `Application Logging`. 

`key` was not changed by this action because split isn't destructive. 

In the string at the bottom, the underscores are replaced with hyphens, to create the `[#application-logging]` bit. 

Is there a different way you'd like me to do this? I could try to take care of it after `key` has been split instead. I don't feel strongly about how it's implemented.

Reply:
Ah, sorry @kaylareopelle - I misread the code as `"#{title} [##{title.tr(' ', '-')}]"`.